### ISSUE TITLE
udiskie/prompt.py gives error on install

### ISSUE THREAD

**Issue by technerd0103** [coldfix/udiskie#190]
error when running program from line 15 in udiskie/prompt.py

current:
```python
from importlib_resources import read_text
```
should be:
```python
from importlib.resources import read_text
```

**Comment by coldfix** [comment]
Hi, thanks for reporting. Should be fixed by #189.

**Comment by mb720** [comment]
Thanks for fixing this.

This was the stack trace, by the way:

    Traceback (most recent call last):
      File "/usr/lib/python3.8/site-packages/udiskie/cli.py", line 223, in _start_async_tasks
        results = await self._init()
      File "/usr/lib/python3.8/site-packages/udiskie/cli.py", line 340, in _init
        import udiskie.prompt
      File "/usr/lib/python3.8/site-packages/udiskie/prompt.py", line 15, in <module>
        from importlib_resources import read_text
    ModuleNotFoundError: No module named 'importlib_resources'
